src/app/detail-page/detail-page.component.html

```diff
@@ -1,7 +1,6 @@
 <div>
   <app-search-bar></app-search-bar>
 
-  <router-outlet></router-outlet>
+  <app-detail-content></app-detail-content>
```

Comment:
I don't think you need extra app-detail-content component, just move the logic of detail-content-component to detail-page.component. In the detail-page.component.html file